addons/account/wizard/account_move_reversal.py

```diff
@@ -26,8 +26,42 @@ class AccountMoveReversal(models.TransientModel):
             ('modify', 'Full refund and new draft invoice')
         ], string='Credit Method', required=True,
         help='Choose how you want to credit this invoice. You cannot "modify" nor "cancel" if the invoice is already reconciled.')
-    journal_id = fields.Many2one('account.journal', string='Use Specific Journal', help='If empty, uses the journal of the journal entry to be reversed.', check_company=True)
+    journal_id = fields.Many2one(
+        'account.journal',
+        check_company=True,
+        default=lambda self: self._get_default_journal_id(),
+        help='If empty, uses the journal of the journal entry to be reversed.',
+        required=True,
+        string='Use Specific Journal',
```

Comment:
this order is more logical to me: `comodel_name`, `string`, `required`, `default`, `check_company`, `help`